plugins/foreman_openscap/0.5/index.md

```diff
@@ -119,7 +122,7 @@ Edit ```/etc/foreman-proxy/settings.d/openscap.yml``` with the appropriate setti
 
     puppet module install isimluk-foreman_scap_client
 
-This puppet module will automatically install foreman_scap_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client. The module is used by Foreman to automate the configuration and trigigering foreman_openscap_client, so you should install it in all puppet environments that your hosts use.
+This puppet module will automatically install foreman\_scap\_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client. The module is used by Foreman to automate the configuration and triggering of foreman_openscap_client, so you should install it in all puppet environments that your hosts use.
```

Comment:
escaping is probably not necessary - http://www.theforeman.org/plugins/foreman_openscap/0.5/index.html#2.3Installingpuppet-foreman_scap_client